tidb-lightning/tidb-lightning-configuration.md

```diff
@@ -234,8 +234,14 @@ delimiter = '"'
 # Line terminator. Empty value means both "\n" (LF) and "\r\n" (CRLF) are line terminators.
 terminator = ''
 # Whether the CSV files contain a header.
-# If `header` is true, the first line will be skipped.
+# If `header` is true, TiDB Lightning treats the first row as a table header and will not import it as data. If `header` is false, the first row is also imported as CSV data.
 header = true
+# Whether the CSV table header matches the structure of the target table.
+# The default value is true, which means that when importing data, TiDB Lightning matches the column name of the CSV table header with the corresponding column of the target table. Even if the order of the columns in the CSV file and the target table is inconsistent, the data can still be imported.
```

Comment:
```suggestion
# The default value is set to "true", which means that TiDB Lightning will match the column names in the CSV header with those in the target table, so that even if the order of the columns is different between the two, the data can still be imported successfully.
```